### ISSUE TITLE
Malformed UTF-8 character (fatal)

### ISSUE THREAD

**Issue by eserte** [kaz-utashiro/App-week#3]
On some of my smokers the test suite fails like this:
```
#   Failed test '1752/9/2'
#   at t/01_run.t line 31.
#          got: 'Malformed UTF-8 character (fatal) at /home/cpansand/.cpan/build/2020032706/Text-ANSI-Fold-1.02-Yt0iRK/blib/lib/Text/ANSI/Fold.pm line 179.
# '
#     expected: '                          
#         September         
#    Su Mo Tu We Th Fr Sa   
#           1  2 14 15 16   
#    17 18 19 20 21 22 23   
#    24 25 26 27 28 29 30   
#                           
#                           
#                           
#                           
# '
# Looks like you failed 1 test of 1.
t/01_run.t ...... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

On one of these systems (freebsd 10) I checked the locale setting, and it was `LC_ALL = de_DE.ISO8859-1` --- so maybe this (a non-utf8 locale) might be the problem.

**Comment by eserte** [comment]
A variation of the failure (maybe with newer perl versions):
```
#   Failed test '1752/9/2'
#   at t/01_run.t line 31.
#          got: 'Malformed UTF-8 character: \xe4\x72\x7a (unexpected non-continuation byte 0x72, immediately after start byte 0xe4; need 3 bytes, got 1) in substitution (s///) at /home/cpansand/.cpan/build/2020032706/Text-ANSI-Fold-1.02-0/blib/lib/Text/ANSI/Fold.pm line 179.
# Malformed UTF-8 character (fatal) at /home/cpansand/.cpan/build/2020032706/Text-ANSI-Fold-1.02-0/blib/lib/Text/ANSI/Fold.pm line 179.
# '
#     expected: '                          
#         September         
#    Su Mo Tu We Th Fr Sa   
#           1  2 14 15 16   
#    17 18 19 20 21 22 23   
#    24 25 26 27 28 29 30   
#                           
#                           
#                           
#                           
# '
# Looks like you failed 1 test of 1.
t/01_run.t ...... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by eserte** [comment]
I don't see this error with 0.04 (maybe even fixed in 0.03).
